audioswitch/src/main/java/com/twilio/audioswitch/selection/AudioDeviceManager.kt

```diff
@@ -12,16 +13,17 @@ import com.twilio.audioswitch.android.LogWrapper
 private const val TAG = "AudioDeviceManager"
 
 internal class AudioDeviceManager(
-    private val context: Context,
-    private val logger: LogWrapper,
-    private val audioManager: AudioManager,
-    private val build: BuildWrapper,
-    private val audioFocusRequest: AudioFocusRequestWrapper
+        private val context: Context,
```

Comment:
Please run `./gradlew spotlessApply` to fix formatting.

Reply:
Done!